clog/config.py

```diff
@@ -86,10 +86,37 @@
     default=True,
     help="Disable writing logs to monk.")
 
+default_backend = clog_namespace.get_string('default_backend',
+    default="scribe",
+    help="The default backend to use (can be 'scribe', 'monk' or 'dual')")
+
+stream_backend = clog_namespace.get_list('stream_backend',
+    default=[],
+    help="The map of stream names to backend ('scribe', 'monk' or 'dual'). "
+    "If not specified, the default one will be used. The mapping must be "
+    "represente as a list using the format\n"
+    "    - stream_name: backend_name"
```

Comment:
What about we group by stream_backend instead of stream? That would make the configuration more readable and reduce the amount of duplicated information. It could be actually even more explicit, something like:
monk_backend_streams:
\- stream1
....
scribe_backend_streams:
\- stream1
....

Reply:
Good idea, will do.

Reply:
Actually, i'm not sure how that would look. What would be the meaning of the "scribe_backend_streams" list be? Streams that only go to scribe? If that's the case, the monk logger should know about this and check the scribe configuration and vice versa. Instead with the scribe/monk/dual setting, every logger just knows about itself.

For example: if a stream is only in the monk list, and the default backend is scribe, where should messages go? If we want them to go to monk only, then the scribe logger should implement some logic such as "if stream not in monk_streams"... which would make the code more complex.

If i correctly understand your proposal, this is how the configuration would look like assuming that stream 1 only goes to scribe, stream 2 goes to both, stream 3 goes to monk and everything else goes to scribe:

```yaml
default_backend: scribe
scribe_backend_streams:
    - stream1    # not needed
    - stream2
monk_backend_streams:
    - stream2
    - stream3
```

```yaml
default_backend: scribe
stream_backends:
    - steam1: scribe   # not needed
    - stream2: dual
    - stream3: monk
```

Reply:
I see your point. I think that makes sense. You can drop this issue.